src/Test/BaseTemplateIntegrationTestCase.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace Twig\Test;
+
+use PHPUnit\Framework\TestCase;
+use Twig\Environment;
+use Twig\Error\Error;
+use Twig\Extension\ExtensionInterface;
+use Twig\Loader\ArrayLoader;
+use Twig\RuntimeLoader\RuntimeLoaderInterface;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+use Twig\TwigTest;
+
+abstract class BaseTemplateIntegrationTestCase extends TestCase
```

Comment:
We don't want to keep this class when the deprecated class `IntegrationTestCase` will be removed. Neither does the `doIntegrationTest` method.

For cleaner code, I'd suggest converting this class into a trait `Twig\Test\Internal\IntegrationTestTrait` with `@internal` annotation and make `doIntegrationTest` private, so that it cannot be used or extended.